spec/services/dialog_local_service_spec.rb

```diff
@@ -77,6 +77,13 @@
                        "cloud_network", "cloud_networks", "/cloud_network"
     end
 
+    context "when the object is a private CloudNetwork" do
+      let(:obj) { double(:class => ManageIQ::Providers::Openstack::NetworkManager::CloudNetwork::Private, :id => 123) }
+
+      include_examples "DialogLocalService#determine_dialog_locals_for_custom_button return value",
+                       "private", "cloud_networks", "/cloud_network"
```

Comment:
@eclarizio does this look right to you? that the target_type gets set to "private" in this case? it looks pretty funky.

Reply:
Yeah, that does seem a bit weird and I don't think it will work because we're basically going to be making a call to /api/service_dialogs/<dialog_id> and passing in `resource_action_id`, `target_id`, and `target_type`. This eventually goes and tries to look up the object by collection name, and there is no collection called `private` in the `config/api.yml` file. I think this would probably need to end up being `cloud_networks`.